tools/ci-cdk/canary-runner/src/build_bundle.rs

```diff
@@ -56,16 +56,19 @@ tracing-texray = "0.1.1"
 reqwest = { version = "0.11.14", features = ["rustls-tls"], default-features = false }
 "#;
 
+const REQUIRED_SMITHY_CRATES: &[&str] = &["aws-smithy-async"];
```

Comment:
If `FnStream` is required to implement event streams now, then we need to re-export it from the generated crates (when generating for event streams).

Reply:
Good point. Canary shouldn't need to directly depend on `aws-smithy-async` that way. Addressed in b281aee.